deploy/internal/deployment-endpoint.yaml

```diff
@@ -49,6 +51,9 @@ spec:
           ports:
             - containerPort: 6001
             - containerPort: 6443
+          envFrom:
```

Comment:
This will load all of the envs from the config map. since we want to have a single configmap for all pods I think it's better to pick specific envs and use `valueFrom`

Reply:
correct, but any new env vars added to the config map will not be loaded to the pods.
I don't know if its a use case we want to cover, if not, we can change to `valueFrom`.